docs/reference/aggregations/pipeline/inference-bucket-aggregation.asciidoc

```diff
@@ -6,13 +6,13 @@
 <titleabbrev>{infer-cap} bucket</titleabbrev>
 ++++
 
-beta::[]
```

Comment:
I am not 100% sure on this one not being beta.

@davidkyle ?

Reply:
No we are GA in the next release 🚀